docs/testing/releases/630.md

```diff
@@ -0,0 +1,143 @@
+4. Confirm you can load the template. It will have a temporary placeholder block. This SVG will be replaced in a future PR.
+5. Confirm the behavior on the frontend product page is identical to the behavior on the product page without this PR applied.
+
+### FSE: Add basic product archive block template. ([5049](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5049))
+
+1. Install and activate [the Gutenberg plugin](https://wordpress.org/plugins/gutenberg/) (not the latest built version of `trunk` from the repo).
+2. Install and activate a FSE enabled theme, such as [the Tove theme](https://wordpress.org/themes/tove/).
+3. Load the Site Editor (Appearance → Editor) and select the `Product Archive Page` template from the `General templates` section.
+4. Confirm you can load the template, and it looks as expected in large/small viewports as per the screenshots above.
+5. Confirm the behavior on the Frontend on `/shop` is identical to the behavior on `/shop` without this PR applied.
+
+### Add placeholder text when modifying product search input in the editor. ([5122](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5122))
+
+1. Add the Product Search block to any page in the editor.
+2. Delete the “Search Products…” value from the Search input.
+3. “Enter search placeholder text” should be displayed in the input as a placeholder.
+4. The text should disappear as you start typing.
+
+### Improve the product images placeholder display by adding a light gray border to it. ([4950](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4950))
+
+1. Create a product with no product image.
+2. Create a test page and add the Newest Products block.Ftext
+3. Verify that the placeholder image has a visible border.
+
+### Improve accessibility by using self-explaining edit button titles. ([5113](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5113))
+
+1. Create a test page and add the following blocks to it:
+    - Featured Category block
+    - Featured Product block
+    - Hand-picked Products block
+    - Products by Category block
+    - Products by Attribute block
+    - Reviews by Product block
+    - Reviews by Category block
+    - Products by Tag block
+    - All products block
+    - Single Product block
+2. Click on the `edit` button of each block and verify that the button text corresponds to the value of the following list:
+    - Featured Category: Edit selected category
+    - Featured Product: Edit selected product
+    - Hand-picked Products: Edit selected products
+    - Products by Category: Edit selected categories
+    - Products by Attribute: Edit selected attributes
+    - Reviews by Product: Edit selected product
+    - Reviews by Category: Edit selected categories
+    - Products by Tag: Edit selected tags
+    - All products: Rearrange inner blocks
```

Comment:
Mine says "Edit inner product layout"

Reply:
Good catch! Based on https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5113#discussion_r746240869, the string had been changed from `Rearrange inner blocks` to `Edit inner product layout` and updated the testing instructions.